packages/app-builder-lib/templates/nsis/README.md

```diff
@@ -12,7 +17,13 @@ http://stackoverflow.com/questions/2565215/checking-if-the-application-is-runnin
 
 One-click installer: http://forums.winamp.com/showthread.php?t=300479
 
-# GUID
+## Localization (l10n)
+
+For translators, the strings to be displayed are included in [`assistedMessages.yml`](https://github.dev/l10n-tw/electron-builder/blob/master/packages/app-builder-lib/templates/nsis/assistedMessages.yml) and [`messages.yml`](https://github.dev/l10n-tw/electron-builder/blob/master/packages/app-builder-lib/templates/nsis/messages.yml).
```

Comment:
Please adjust these links so that they reference the electron-builder repo :)
https://github.com/electron-userland/electron-builder/blob/master/packages/app-builder-lib/templates/nsis/assistedMessages.yml
https://github.com/electron-userland/electron-builder/blob/master/packages/app-builder-lib/templates/nsis/messages.yml

Reply:
Sorry for the careless fault. The links reference to this repository now.